src/control/cmd/dmg/pretty/pool.go

```diff
@@ -303,17 +310,39 @@ func printListPoolsRespVerbose(out io.Writer, resp *control.ListPoolsResp) error
 			continue
 		}
 		table = append(table, poolListCreateRowVerbose(pool))
+
 	}
 
 	fmt.Fprintln(out, formatter.Format(table))
 
 	return nil
 }
 
+// UpdateListPoolsResponse will update the pool which has been rebuild and not in idle state
+func UpdateListPoolsResponse(final_resp *control.ListPoolsResp, resp *control.ListPoolsResp, rebuildOnly bool) error {
+	// Update the pool state to Degraded if any pool target is disabled.
+	for _, pool := range resp.Pools {
+		if pool.TargetsDisabled > 0 {
+			pool.State = fmt.Sprintf("%s-%s", pool.State, "Degraded")
+		}
```

Comment:
This is already done in the server-side changes, so should be removed here. I think server side is the right place for it.